chore: update tutorials and docs to use MBridge instead of NeMo2 format (#677)
NVIDIA-NeMo/Evaluator: docs/deployment/nemo-fw/pytriton.md

```

When working with a larger model, you can use model parallelism to distribute the model across available devices.

> In the example below we deploy the [Llama-3_3-Nemotron-Super-49B-v1](https://huggingface.co/nvidia/Llama-3_3-Nemotron-Super-49B-v1) (converted to the Megatron Bridge format) with 8 devices and tensor parallelism:

Reviewer: ```suggestion
The example below deploys the [Llama-3_3-Nemotron-Super-49B-v1](https://huggingface.co/nvidia/Llama-3_3-Nemotron-Super-49B-v1) model (converted to the Megatron Bridge format) with 8 devices and tensor parallelism:
```
